### DISCUSSION TITLE
version 1.6.6 and no more link to view the archived document

### DISCUSSION THREAD

**Discussion by tradexsrl** [docusealco/docuseal#328] [Q&A]
Hi , in this version if you have this view:
![immagine](https://github.com/user-attachments/assets/8f301b95-1c55-4cf2-90d1-66d27ec3862a)
you have no more the link to view the archived doc
if you switch to:
![immagine](https://github.com/user-attachments/assets/9920d309-7840-459e-81f6-6a2e6e27e5ff)
you have it
![immagine](https://github.com/user-attachments/assets/d0712295-5848-4c9f-8551-fd700969bab2)
please add it also in the other view
thank's

**Comment by omohokcoj** [comment]
@tradexsrl on the templates view you can switch to see the archived templates (but not submissions) - looks like in your case there are no archived templates that's why the link is not displayed there

**Comment by tradexsrl** [comment]
undesrtand , maybe the bug was in the previous version where in template i saw also archived link but are not archived template.
thank's
